io.itch.amcsquad.amcsquad.metainfo.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component>
+  <id type="desktop">io.itch.amcsquad.amcsquad</id>
+  <launchable type="desktop-id">io.itch.amcsquad.amcsquad.desktop</launchable>
+  <name>The AMC Squad</name>
+  <developer>AMC Team</developer>
```

Comment:
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#name-summary-and-developer-name